app/Http/Controllers/DemandController.php

```diff
@@ -101,7 +100,7 @@ public function viewScript(Request $request)
     {
         $params = [json_encode($request->getSchemeAndHttpHost())];
 
-        $jsPath = config('app.env') == 'production' ? resource_path('assets/js/tmp-copy/view.x.js') : resource_path('assets/js/tmp-copy/view.min.js');
+        $jsPath = public_path('-/view.js');
 
         $response = new StreamedResponse();
         $response->setCallback(function () use ($jsPath, $request, $params) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$request" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1481)